controllers/grid/users/reviewer/form/ReviewerForm.inc.php

```diff
@@ -241,12 +242,16 @@ function fetch($request) {
 			'reviewDueDate' => __('reviewer.submission.reviewDueDate'),
 			'submissionReviewUrl' => __('common.url'),
 			'reviewerUserName' => __('user.username'),
+			'contextName' => $context->getLocalizedName(),
+			'contextUrl' => __('common.url'),
+			'editorialContactSignature' => strip_tags($user->getContactSignature(), "<br>"),
```

Comment:
I was not familiar with the convention here, so I followed how `SubmissionMailTemplate` handles these variables.

https://github.com/pkp/pkp-lib/blob/cb6fa16ad663618fe2133dc6dec556bb0e8535fa/classes/mail/SubmissionMailTemplate.inc.php#L51-54